common/datavalues2/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "common-datavalues2"
+version = "0.1.0"
+authors = ["Databend Authors <opensource@datafuselabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[lib]
+doctest = false
+test = false
+
+[dependencies] # In alphabetical order
```

Comment:
It looks like some of the dependencies aren't actually used?

Reply:
Yes，is there a way to detect which deps are not used in editor?

Reply:
None. Even udeps do not work. :cry: